tailwind.config.js

```diff
@@ -14,6 +14,17 @@ module.exports = {
         'sky': {
           650: '#0575E6'
         },
+        'main': {
+          100: '#111111',
+          200: '#FFFFFF',
+          400: '#E1E1E1',
+          500: '#7A7A7A',
```

Comment:
ესე ამათი გაწერა არაფერში გამოსადეგია, მე რო ვუყურებ რავიცი რას გულისხომ 100 200 300, შეიძლება შავი წითელი ან ნებისმიერი იყოს, მომვალისთვი წერე როგორც გითხარი.

Reply:
[✔] - Tell me how to named them

Reply:
https://find-nearest-tailwind-colour.netlify.app/
აქ ჩაწერე ფერი და რა სახელსაც ამოაგდებს მაგალითად : red-500 დაარქვი ახლოს მაგასთან red-550 ან red-530 როგორც დაგჭირდება თუ 500 600ს შორის ბერვი იყო დაარქვი 510 520 ...